src/projects.h

```diff
@@ -226,12 +226,18 @@ typedef struct ARG_list {
 	/* base projection data structure */
 
 
+#ifdef PJ_LIB__
+    /* we need this forward declaration in order to be able to add a
+       pointer to struct opaque to the typedef struct PJconsts below */
+    struct opaque;
```

Comment:
Forget my above comment. The #ifdef PJ_LIB__ likely only exposes that to proj internals and not user code.